.github/workflows/npm-sdk-publish.yml

```
@@ -0,0 +1,32 @@
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 22
+          registry-url: https://npm.pkg.github.com
+          cache: yarn
+      - run: |
+          rm .npmrc
+          npm config set ${REGISTRY}:_authToken ${{ secrets.NPM_PACKAGE_TOKEN }}
+        env:
+          REGISTRY: //registry.npmjs.org/
+      - run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:2:15: Double quote to prevent globbing and word splitting

(shellcheck)